Sources/SimultaneouslyScrollView/DefaultSimultaneouslyScrollViewHandler.swift

```diff
@@ -13,46 +23,63 @@ internal class DefaultSimultaneouslyScrollViewHandler: NSObject, SimultaneouslyS
     }
 
     func register(scrollView: UIScrollView) {
-        guard !scrollViewsStore.contains(scrollView) else {
+        register(scrollView: scrollView, for: .none)
+    }
+
+    func register(scrollView: UIScrollView, for scrollDirection: ScrollDirection?) {
```

Comment:
Do we need the overload for non- API breaking changes? 🤔 It should be possible to add the `for scrollDirection: ScrollDirection? = nil` to the existing `register(...)` methods AFAIK.

Reply:
Turns out since we use a protocol with the overload we need the overload here as well